app/serializers/site_serializer.rb

```diff
@@ -22,4 +22,11 @@ class SiteSerializer < ActiveModel::Serializer
 
   has_many :categories
   has_many :links
+
+  has_many :recent_posts do
+    link :related do
+      api_site_posts_path(object.fqdn, page: {size: 5})
```

Comment:
I decided to not include this link.
`/posts` with `{page: {number: 5}}` is enough implementation at this time.